exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/model/MetadataProvider.java

```diff
@@ -38,17 +37,16 @@
   TupleMetadata tuple();
   VariantMetadata variant();
 
-  public static class VectorDescrip {
+  class VectorDescrip {
```

Comment:
public? Or, if intentionally protected, is handy to declare that so we know that protected is intentional.

Also, the class changed from static to inner: this means that it must be created in the context of its outer class, and carries a pointer to the outer object. Since that was not needed here, the original declaration used "static" to avoid unnecessary complexity.